tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "noEmit": true,
+    "module": "esnext",
+    "moduleResolution": "Bundler",
+    "lib": ["esnext"],
+    "strict": true,
+    "types": ["@cloudflare/workers-types/2022-03-21"],
```

Comment:
This path is set based on the compatibility date in wrangler.toml. If we update that compatibility date, this should also be updated as described in https://github.com/cloudflare/workerd/tree/main/npm/workers-types#compatibility-dates